polytope/polytope.py

```diff
@@ -78,7 +78,10 @@
     logger.warn(
         '`polytope` failed to import `cvxopt.glpk`.\n'
         'Will use `scipy.optimize.linprog`.')
-
+try:
+    xrange
```

Comment:
The `try`-`except` block is good. I was mostly curious about time costs of the `hasattr()` check vs. the no-op usage of `xrange`, but I am not aware that module imports would occur frequently enough for it to matter.